### PR TITLE
Feature/migrations Alembic Async Migrations.

### PR DESCRIPTION
**Pull request by Jsyro**
Alembic controlled migrations
- auto upgrade on startup
Async psql connections/sessions
local development hot reloading

sqlalchemy.future.select is apparently the very 'new school' stuff in sqlalchemy 2.0. lets do more digging. but `session.query(cls)` doesn't work with async sessions.

### CODE REVIEW

File: services/traction/README.md [hunk 1]
```diff
@@ -3,9 +3,16 @@ pip install -r requirements.txt
...
 pip install tox
 tox -e lint
 tox -e test
-```
\ No newline at end of file
+```
+
+how to generate new alembic revision
+
+```docker-compose exec scripts_traciton-api_1 alembic --revision --autogenerate -m "<MIGRATION COMMENT HERE>"
```
Comment (ianco) [inline services/traction/README.md:14]: Spelling?

**Review by ianco — CHANGES_REQUESTED** [review]
Minor spelling

File: services/traction/README.md (hunk 1 above)
Comment (Jsyro) [reply services/traction/README.md]: there were multiple issues, this command now actually works. :)

**Review by ianco — APPROVED** [review]
lgtm

**Pull request merged**